service/frontend/adminHandler.go

```diff
@@ -1431,7 +1432,9 @@ func (adh *AdminHandler) ResendReplicationTasks(
 }
 
 // GetTaskQueueTasks returns tasks from task queue
-// TODO: support pagination
+// TODO: existing behavior for the requested range is (exclusive, inclusive],
+// we can deprecate old fields and define new one to make the range consistent
+// with the persistence layer.
```

Comment:
I think it is ok to break this. This API is internal and only used by tctl. There is no clear contract that the requested min/max is inclusive or exclusive.

Reply:
Sounds good. Removed the backward compatible logic and updated CLI usage.